src/containers/tutor-home-page/general-info-step/GeneralInfoStep.tsx

```diff
@@ -62,23 +97,25 @@ const GeneralInfoStep = ({ btnsBox, isUserFetched, setIsUserFetched }) => {
     }
   }
 
-  const onChangeCity = (_, value) => {
-    handleNonInputValueChange('city', value)
+  const onChangeCity = (
+    event: SyntheticEvent<Element, Event>,
+    value: unknown
```

Comment:
if you decide to change for 'options: string[]', I guess, you may change for 'value: string | null' here.